src/document.js

```diff
@@ -307,6 +308,15 @@ const documentsMain = {
 
 						if (Config.get('userId') === null) {
 							PostMessages.sendWOPIPostMessage('loolframe', 'Hide_Menu_Item', { id: 'insertgraphicremote' })
+						} else {
+							PostMessages.sendWOPIPostMessage('loolframe', 'Insert_Button', {
+								id: 'Open_Local_Editor',
+								imgurl: 'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjRweCIgdmlld0JveD0iMCAwIDI0IDI0IiB3aWR0aD0iMjRweCIgZmlsbD0iIzAwMDAwMCI+PHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPjxwYXRoIGQ9Ik0xOSAxOUg1VjVoN1YzSDVjLTEuMTEgMC0yIC45LTIgMnYxNGMwIDEuMS44OSAyIDIgMmgxNGMxLjEgMCAyLS45IDItMnYtN2gtMnY3ek0xNCAzdjJoMy41OWwtOS44MyA5LjgzIDEuNDEgMS40MUwxOSA2LjQxVjEwaDJWM2gtN3oiLz48L3N2Zz4=',
```

Comment:
Maybe we could use something like https://github.com/bhovhannes/svg-url-loader to still have a .svg file in the repo and just build the data representation during build, but not a blocker ;)

Reply:
Changed to use the `imagePath` function. 

Realised why it didnt work before, `imagePath` generates a relative path which would open from the collabora hostname. Made sure to prepend the NC instance url :)